.github/workflows/ci.yaml

```
@@ -18,6 +18,18 @@ jobs:
           path: ~/.cache/pip
           key: ${{ hashFiles('pyproject.toml') }}
       - uses: pre-commit/action@v2.0.3
+
+  Type-Check:
+    runs-on: ubuntu-latest
+    steps:
```

```suggestion
    env:
      MYPY_FORCE_COLOR: 1
      TERM: xterm-color
    steps:
```

This will enable colored output